index.js

```diff
@@ -3,7 +3,8 @@ var util = require('gulp-util')
   , aws = require('aws-sdk');
 
 module.exports = function (options) {
-  options.wait = (options.wait === undefined || !options.wait) ? false : true;
+  options.wait = !!options.wait;
+  options.indexRoot = !!options.indexRoot;
```

Comment:
Defaults to false.  No need to use a conditional only to return a boolean if/else.  Just return the conditional.